app/models/collection.rb

```diff
@@ -106,6 +106,18 @@ def custom_rights_instructions_source_option
     end
   end
 
+  def effective_custom_rights_instructions
+    unless custom_rights_statement_source_option == "entered_by_depositor"
+      raise "Custom rights for #{collection.id} not entered by depositor; thus it doesn't make sense to determine instructions for entering"
+    end
+
+    if custom_rights_instructions_source_option == "provided_by_collection"
+      custom_rights_statement_custom_instructions
+    else
+      Settings.access.default_instructions_for_custom_use_statement
+    end
+  end
+
```

Comment:
similarly, this method felt more like it was entirely about whether anything should care about a particular field value for the model, but could see an argument for putting it in the component.